.clang-format

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
If you want to add `.clang-format`, please in separate commit.

Reply:
It already is a seperate commit ?

Reply:
I stand corrected.